cmd/kind/load/image-archive/image-archive.go

```diff
@@ -96,17 +103,50 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 			selectedNodes = append(selectedNodes, node)
 		}
 	}
+	return loadImageOnNodes(imageTarPath, selectedNodes)
+}
+
+// loadImageOnNodes loads concurrently an image tarball in a group of nodes
+func loadImageOnNodes(imageTarName string, nodes []clusternodes.Node) error {
+	// Load the image into every node concurrently
+	// create all of the node containers, concurrently
+	errChan := make(chan error, len(nodes))
+	var wg sync.WaitGroup
+	for _, node := range nodes {
+		node := node // capture loop variable
+		// Don't load image on external nodes like the load balancer
+		if strings.Contains(node.String(), "external") {
```

Comment:
this is a hack that will bite us later. Should be using the node role

Reply:
done